### PR TITLE
Migration to Play 2.7

### PR DESCRIPTION
**Pull request by mtrushin**
Migration to Play 2.7.0

### CODE REVIEW

**Comment by mtrushin** [comment]
👍

File: app/org/zalando/hutmann/authentication/User.scala [hunk 1]
```diff
@@ -43,11 +43,11 @@ object User {
   * Intermediate user that will only be used internally. Do not use this one directly, use App2AppUser instead.
   */
 private final case class IntermediateUser(
-  accessToken: String,
-  scope:       Set[String],
-  realm:       String,
-  tokenType:   String,
-  expiresIn:   Int
+    accessToken: String,
```
Comment (ovolynets) [inline app/org/zalando/hutmann/authentication/User.scala]: I could imagine the formatting isn't always consistent and in this case later it is padded with two spaces. Do you see specific needs for formatting it differently?

**Comment by ovolynets** [comment]
@mtrushin thanks for the pull request. I put one minor comment but otherwise the changes look good from my side. Would be also nice to have a review from someone who is familiar with Play 2.7 to have a look.

File: app/org/zalando/hutmann/authentication/User.scala (hunk 1 above)
Comment (mtrushin) [reply app/org/zalando/hutmann/authentication/User.scala]: the reason of new formatting is the following change:
screenshot 2019-02-13 at 14 08 33

With new scalariform version `doubleIndentClassDeclaration` got deprecated and we need to use `doubleIndentConstructorArguments` instead. `doubleIndentConstructorArguments: true` is [recommended by the Scala Style Guide](https://docs.scala-lang.org/style/declarations.html#classes), so the result should now be:
```
class Person(
    name: String,
    age: Int,
    astrologicalSign: String)
  extends Entity
  with Logging {
}
```

Before it was:
```
class Person(
  name: String,
  age: Int,
  astrologicalSign: String)
    extends Entity
    with Logging {
}
```

**Comment by mtrushin** [comment]
thanks @ovolynets for the review! I'm afraid it is not easy to find developers familiar with Play 2.7 because it was released only 12 days ago, here is the Migration Guide: https://www.playframework.com/documentation/2.7.x/Migration27

File: app/org/zalando/hutmann/authentication/User.scala (hunk 1 above)
Comment (ovolynets) [reply app/org/zalando/hutmann/authentication/User.scala]: Ah, so you run the formatter? If so it's great. Keep it then

**Comment by ovolynets** [comment]
@mtrushin I'm happy to merge and publish (if I still can). Did you try to integrate these changes to one of your services?

**Comment by ovolynets** [comment]
Ah, no, maybe I can't. @musiKk do you have access to publishing?

File: app/org/zalando/hutmann/authentication/User.scala (hunk 1 above)
Comment (mtrushin) [reply app/org/zalando/hutmann/authentication/User.scala]: I only changed scalariform config, formatting is done on sbt build

**Comment by mtrushin** [comment]
@ovolynets yes, new version is already integrated with one of our services which we want to migrate to Play 2.7, waiting for review now...

**Comment by ovolynets** [comment]
👍

**Comment by ovolynets** [comment]
@mtrushin ok, thanks for clarifying. I'd wait for the admin's approval (@musiKk are you available?).

**Comment by musiKk** [comment]
I'll see if I find time to look into this on Monday. Spoiler alert, I'm definitely not familiar with 2.7. We're still running on 2.5 and will migrate to 2.6 shortly.

**Comment by musiKk** [comment]
:+1:

**Pull request merged**